src/index.js

```diff
@@ -65,7 +65,7 @@ export function sampleRUM(checkpoint, data = {}) {
       const sendPing = (pdata = data) => {
         // eslint-disable-next-line object-curly-newline, max-len, no-use-before-define
         const body = JSON.stringify({ weight, id, referer: window.hlx.rum.sanitizeURL(), checkpoint, t: (Date.now() - firstReadTime), ...data }, knownProperties);
-        const url = `https://rum.hlx.page/.rum/${weight}`;
+        const url = new URL(`.rum/${weight}`, window.location).href;
```

Comment:
Is there a `/` missing ?
From Chrome console I get 
> new URL(`.rum/1`, window.location).href;
'https://github.com/adobe/helix-rum-js/pull/98/.rum/1'

![Screenshot 2023-09-20 at 15 18 53](https://github.com/adobe/helix-rum-js/assets/68709/5a98666f-1788-40be-b6e6-90f1421c65c6)